common/static/common/templates/discussion/discussion-home.underscore

```diff
@@ -15,43 +15,43 @@
     </span>
     <table class="home-helpgrid">
       <tr class="helpgrid-row helpgrid-row-navigation">
-        <td class="row-title"><%- gettext("Find discussions") %></td>
+        <th scope="row" class="row-title"><%- gettext("Find discussions") %></td>
         <td class="row-item">
-          <i class="icon fa fa-reorder"></i>
-          <span class="row-description"><%- gettext("Focus in on specific topics") %></span>
+          <span class="icon fa fa-reorder" aria-hidden="true"></span>
+          <span class="row-description"><%- gettext("Use the Discussion Topics menu to find specific topics.") %></span>
         </td>
         <td class="row-item">
-          <i class="icon fa fa-search"></i>
-          <span class="row-description"><%- gettext("Search for specific posts") %></span>
+          <span class="icon fa fa-search" aria-hidden="true"></span>
+          <span class="row-description"><%- gettext("Search all posts") %></span>
         </td>
         <td class="row-item">
-          <i class="icon fa fa-sort"></i>
-          <span class="row-description"><%- gettext("Sort by date, vote, or comments") %></span>
+          <span class="icon fa fa-sort" aria-hidden="true"></span>
+          <span class="row-description"><%- gettext("Filter and sort topics") %></span>
         </td>
       </tr>
       <tr class="helpgrid-row helpgrid-row-participation">
-        <td class="row-title"><%- gettext("Engage with posts") %></td>
+        <th scope="row" class="row-title"><%- gettext("Engage with posts") %></td>
         <td class="row-item">
-          <i class="icon fa fa-plus"></i>
-          <span class="row-description"><%- gettext("Upvote posts and good responses") %></span>
+          <span class="icon fa fa-plus" aria-hidden="true"></span>
+          <span class="row-description"><%- gettext("Vote for good posts and responses") %></span>
         </td>
         <td class="row-item">
-          <i class="icon fa fa-flag"></i>
-          <span class="row-description"><%- gettext("Report Forum Misuse") %></span>
+          <span class="icon fa fa-flag" aria-hidden="true"></span>
+          <span class="row-description"><%- gettext("Report abuse, topics, and responses") %></span>
```

Comment:
I would suggest "Report misuse" instead of "abuse", and I'm not sure whay the sewquence of abuse, toptics, and responses indicates. Should this be "Report misuse in posts, responses, or comments"?